src/Components/Users/UserProfile.tsx

```diff
@@ -774,35 +802,74 @@ export default function UserProfile() {
                           error={changePasswordErrors.old_password}
                           required
                         />
-                        <TextFormField
-                          name="new_password_1"
-                          label="New Password"
-                          type="password"
-                          value={changePasswordForm.new_password_1}
-                          className="col-span-6 sm:col-span-3"
-                          onChange={(e) =>
-                            setChangePasswordForm({
-                              ...changePasswordForm,
-                              new_password_1: e.value,
-                            })
-                          }
-                          error=""
-                          required
-                        />
-                        <TextFormField
-                          name="new_password_2"
-                          label="New Password Confirmation"
-                          className="col-span-6 sm:col-span-3"
-                          type="password"
-                          value={changePasswordForm.new_password_2}
-                          onChange={(e) =>
-                            setChangePasswordForm({
-                              ...changePasswordForm,
-                              new_password_2: e.value,
-                            })
-                          }
-                          error={changePasswordErrors.password_confirmation}
-                        />
+                        <div className="col-span-6 sm:col-span-3">
+                          <TextFormField
+                            name="new_password_1"
+                            label="New Password"
+                            type="password"
+                            value={changePasswordForm.new_password_1}
+                            className="col-span-6 sm:col-span-3"
+                            onChange={(e) => {
+                              setChangePasswordForm({
+                                ...changePasswordForm,
+                                new_password_1: e.value,
+                              });
+                            }}
+                            required
+                            onFocus={() => setNewPasswordInputInFocus(true)}
+                            onBlur={() => setNewPasswordInputInFocus(false)}
+                          />
+                          {newPasswordInputInFocus && (
```

Comment:
@r-nikhilkumar consider using CSS instead of creating a state 

https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state

Please do the same for confirm password too